src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -419,7 +419,7 @@ public double getLoadTargetQ() {
 
     @Override
     public double getMaxP() {
```

Comment:
It seems to be used only here:
![image](https://github.com/powsybl/powsybl-open-loadflow/assets/48123713/ed2f86b8-1647-4db4-94f3-12fb5651f8ec)
So we prefer to use the generator min and max P restrictions here?

Reply:
@annetill I also think it is onnly used here ans this is the only justification to add minTargetP and maxTargetP in LFGenerator instead of only using minP/maxP that would be overriden.

To really know what to do here, if you are unsure of my interpretation, maybe be should ask the features author or someone who uses that feature. I assumed that the intention is to not start voltage control if the generator is outside of the generator limits minP/macP and not outside of the authorized limites to move targetP (minTargetP/maxTargetP).

Reply:
@geofjamg what do we have to do here when we force to keep one PV bus in the network?